framework/assets/yii.gridView.js

```diff
@@ -237,7 +237,8 @@
      * @param {string} selector jQuery selector
      * @param {function} callback The actual function to be executed with this event
      */
-    function initEventHandler($gridView, type, event, selector, callback) {
+    function initEventHandler($gridView, type, event, selector, callback)
```

Comment:
I thing JS should be validated as well. If not by the `PHPCS` then by what? Don't mind dropping `JS` validation via `PHPSC` altogether though.